### PR TITLE
Docker.

### PR DESCRIPTION
**Pull request by nepeat**
Why not!

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 aiohttp==2.2.5
 async-timeout==2.0.0
 chardet==3.0.4
-discord.py==1.0.0a1393+98a054e
+https://github.com/Rapptz/discord.py/archive/master.zip?1#egg=discord.py[voice]
```
Comment (hedgehog1029) [inline requirements.txt]: we're not using [voice] features right now, so this should just be the standard egg

**Review by hedgehog1029 — DISMISSED** [review]
but why

File: requirements.txt (hunk 1 above)
Comment (nepeat) [reply requirements.txt]: voice stuff planned for later, installing by requirements.txt fails with that version string

also i love docker

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+version: '2'
+services:
+  redis:
```
Comment (hedgehog1029) [inline docker-compose.yml:3]: what the fuck?

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,7 @@
+DEPLOY=DEVELOPMENT
```
Comment (hedgehog1029) [inline .env.example:1]: not entirely sure what these values are?

**Review by hedgehog1029 — CHANGES_REQUESTED** [review]
can u fuckin not

**Comment by hedgehog1029** [comment]
this isn't getting merged to master because it's unneeded for production
you may fork the repo and keep it there if you so wish

**Pull request closed without merging**